frontend/test/pytest/test_pytree_args.py

```diff
@@ -219,7 +219,7 @@ def else_fn():
     def test_return_value_dict(self, backend):
         """Test dictionaries."""
 
-        @qml.qnode(qml.device(backend, wires=2))
+        @qml.qnode(qml.device(backend, wires=2, shots=1000))
```

Comment:
Is there a reason to convert an existing test from analytic to finite-shots?

Reply:
Fixed!